CRT.c

```diff
@@ -1001,23 +1003,41 @@ void CRT_setMouse(bool enabled) {
 }
 #endif
 
-static bool termKeySupport(const char* termType) {
-   if (
-      String_startsWith(termType, "xterm") ||
-      String_eq(termType, "vt220") ||
-      String_startsWith(termType, "tmux") ||
-      /* optimize condition checking by short circuiting logic  */
-      (termType[0] == 's' && (
-         String_startsWith(termType, "screen") ||
-         String_eq(termType, "st") ||
-         String_startsWith(termType, "st-")
-      )) ||
-      String_eq(termType, "alacritty")
-   ) {
-      return true;
-   } else {
+static bool termSupportsDefinedKeys(const char* termType) {
+   if (!termType) {
       return false;
    }
+   switch (termType[0]) {
```

Comment:
Blank line before.

Also the `break;` should be indented … Compare with other `switch` statements or the style guide …

Reply:
@unrealapex An informative note: the `break` statement is part of the `case` block. A `case` block doesn't need to be terminated by a `break` statement, and there might be multiple `break` statements if the block has multiple exit points. That's the reason the `break` statement is indented one level below the `case` label. It looks like an elementary mistake to me if someone doesn't indent the `break` statement in that usual way.